Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml

```diff
@@ -12,8 +12,7 @@ http://www.springframework.org/schema/aop     http://www.springframework.org/sch
         The beans available in this file must be identified in the "description" element for the Spring configuration application. Beans are listed between braces
```

Comment:
Per offline discussion with Jason: this is not specifically a "POM" fix, but it is a build fix -- with such a large overlap between two concepts, this fix will be left in.  I have added a brief justification to the PR description.